manager/integration/tests/test_basic.py

```diff
@@ -321,85 +321,94 @@ def test_backup(clients, volume_name):  # NOQA
 
 
 def backup_test(client, host_id, volname):
-    volume = client.by_id_volume(volname)
-
     setting = client.by_id_setting("backupTarget")
-    setting = client.update(setting, value=common.get_backupstore_url())
-    assert setting["value"] == common.get_backupstore_url()
-
-    if is_backupTarget_s3(setting["value"]):
-        credential = client.by_id_setting("backupTargetCredentialSecret")
-        credential = client.update(credential,
-                                   value=common.get_backupstore_credential())
-        assert credential["value"] == common.get_backupstore_credential()
-
-    volume.snapshotCreate()
-    snap2 = volume.snapshotCreate()
-    volume.snapshotCreate()
+    # test backupTarget for multiple settings
+    backupstores = common.get_backupstore_url()
```

Comment:
Move the setting of `backupTarget` and credential to `test_backup` before calling `backup_test` should make it easier and we won't need so many indent changes (and the change of logic will be hard to spot in the diff).

Btw, can rename `setting` to e.g. `backuptarget_setting`.